src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getComments(postId: number) {
+  const response = await fetch(`${BASE_URL}/comments?postId=${postId}`);
+
+  if (!response.ok) {
+    throw new Error(`${response.status} - ${response.statusText}`);
+  }
+
+  return response.json();
+}
+
+export async function delComment(id : number) {
```

Comment:
Don’t shorten variables or function names. Check the [doc](https://mate-academy.github.io/style-guides/javascript.html#2311-dont-shorten-variables-or-functions-names).
Provide an explicit return type in a function declaration.
```suggestion
export async function deleteComment(id : number): Promise<number> {
```